infra/provider.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.80.0"
+    }
+  }
+
+  cloud {
+    organization = "ACIA-CFIA"
```

Comment:
if we want to allow non-CFIA deployment, I'm guessing this should be a variable/parameter?